bower.json

```diff
@@ -2,19 +2,19 @@
   "name": "ember-cli-materialize",
   "dependencies": {
     "blanket": "~1.1.5",
-    "ember": "1.13.2",
+    "ember": "v2.0.3",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
     "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
     "ember-qunit": "0.4.9",
     "ember-qunit-notifications": "0.0.7",
     "ember-resolver": "~0.1.20",
-    "jquery": "^1.11.3",
+    "jquery": "2.1.4",
     "loader.js": "ember-cli/loader.js#3.2.1",
-    "materialize": "~0.97.0",
+    "materialize": "0.97.5",
     "qunit": "~1.18.0"
   },
   "resolutions": {
-    "jquery": "^1.11.3"
+    "ember": "v2.0.3"
```

Comment:
see above RE: formatting of version targets. You shouldn't need to resolve a bower version conflict either.